.github/workflows/ci.yml

```diff
@@ -0,0 +1,55 @@
+name: CI
+
+on: [push, pull_request]
+
+jobs:
+  format:
+    name: Format
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1
+
+      - name: Install OTP and Elixir
+        uses: ericmj/setup-elixir@14bacb3
+        with:
+          otp-version: 22.1
+          elixir-version: 1.9.4
+
+      - name: Install dependencies
+        run: mix deps.get
```

Comment:
We don't need to install deps for `mix format` I believe, it'll just format looking at `.formatter.exs`. I might have missed this in the Mint setup.

Reply:
Very good point. It is needed for mint because of `import_deps: [:stream_data]`.

Reply:
Ah, didn't think about that. Awesome!